Tasks/VsTest/task.json

```diff
@@ -3,21 +3,22 @@
     "name": "VSTest",
     "friendlyName": "Visual Studio Test",
     "description": "Run tests with Visual Studio test runner",
-    "helpMarkDown": "[More Information](https://go.microsoft.com/fwlink/?LinkId=624539)",
+    "helpMarkDown": "[More Information](https://go.microsoft.com/fwlink/?LinkId=835764)",
     "category": "Test",
     "visibility": [
         "Build",
         "Release"
     ],
     "author": "Microsoft Corporation",
     "version": {
-        "Major": 1,
+        "Major": 2,
         "Minor": 0,
-        "Patch": 81
+        "Patch": 0
     },
     "demands": [
         "vstest"
     ],
+    "preview": "true",
```

Comment:
you also need releaseNotes element which contains markdown.  That field offers a description of what this new major version offers.  It is advertised in the web UI for why this customer should try this out.  I think you would say test impact and minimatch find support.